etcd-aws-cluster

```diff
@@ -16,6 +16,12 @@ server_port=${ETCD_SERVER_PORT:-2380}
 add_ok=201
 already_added=409
 delete_ok=204
+delete_gone=410
```

Comment:
I went digging through the code and although the documentation doesn't mention it, 410 is a valid response: https://github.com/coreos/etcd/blob/66e5e4f298e66bdd72307336935e192aa66097c0/etcdserver/api/v2http/client.go#L276-L277 

I think it should be fine to leave as is.